src/components/lifetime/HeroSection.tsx

```diff
@@ -61,7 +63,11 @@ const HeroSection = ({
               </div>
               <button
                 onClick={() => {
-                  window.location.href = `#payment`;
+                  if (isComeback) {
```

Comment:
instead of passing a boolean prop `isComeback`, pass a function, like `onRedirectButtonClicked` with the redirection for each case.